lib/Number/Compare.pm

```diff
@@ -62,28 +70,29 @@ Now this would be very pointless, if Number::Compare didn't understand
 magnitudes.
 
 The target value may use magnitudes of kilobytes (C<k>, C<ki>),
-megabytes (C<m>, C<mi>), or gigabytes (C<g>, C<gi>).  Those suffixed
-with an C<i> use the appropriate 2**n version in accordance with the
+megabytes (C<m>, C<mi>), gigabytes (C<g>, C<gi>), terabytes (C<t>, C<ti>),
+petabytes (C<p>, C<pi>), and exabytes (C<e>, C<ei>).
+Those suffixed with an C<i> use the appropriate 2**n version in accordance with the
 IEC standard: http://physics.nist.gov/cuu/Units/binary.html
 
 =head1 METHODS
 
-=head2 ->new( $test )
+=head2 -E<gt>new( $test )
```

Comment:
Are these pod changes (all changes of `>` to `E<gt>` and `<` to `E<lt>`) really needed?  

They're not ambiguous and render correctly using both the perldoc and pod2html of perl 5.18.4 and 5.34.0

Reply:
Some pod tools apparently don't always handle -> correct. The perlpodspec doc says

    Compliant parsers must not treat "->" as special.

I ran the pod through podlint, which said that these could cause problems.

Reply:
Do you want me to revert these, or leave them with the escapes in?
I've pushed an update with the other changes.